pkg/load/load.go

```diff
@@ -186,22 +234,42 @@ func Config(path, unresolvedPath, registryPath string, info *ResolverInfo) (*api
 	return &configSpec, nil
 }
 
-func configFromResolver(info *ResolverInfo) (*api.ReleaseBuildConfiguration, error) {
+func configFromResolver(info *ResolverInfo, options configLoadOpt) (*api.ReleaseBuildConfiguration, error) {
+	queryParams := map[string]string{
+		"org":    info.Org,
+		"repo":   info.Repo,
+		"branch": info.Branch,
+	}
 	identifier := fmt.Sprintf("%s/%s@%s", info.Org, info.Repo, info.Branch)
 	if info.Variant != "" {
 		identifier = fmt.Sprintf("%s [%s]", identifier, info.Variant)
```

Comment:
Isn't this logic (and the version on the injected one) already a method on `api.Metadata` ? Why doesn't the `ResolverInfo` embed one?

Reply:
But it doesn't seem this logic is a method on api.Metadata